### ISSUE TITLE
Dependency Issue: 'sklearn'

### ISSUE THREAD

**Issue by lfpelison** [evidentlyai/evidently#17]
Hi, Evidently Team! Thanks for the great package! Very very useful.

I had an issue installing `evidently==0.1.14.dev0`:

```python
Traceback (most recent call last):
  File "main.py", line 8, in <module>
    from evidently.dashboard import Dashboard
  File "/usr/local/lib/python3.6/site-packages/evidently/dashboard/__init__.py", line 4, in <module>
    from .dashboard import Dashboard
  File "/usr/local/lib/python3.6/site-packages/evidently/dashboard/dashboard.py", line 17, in <module>
    from evidently.tabs.base_tab import Tab
  File "/usr/local/lib/python3.6/site-packages/evidently/tabs/__init__.py", line 8, in <module>
    from .classification_performance_tab import ClassificationPerformanceTab
  File "/usr/local/lib/python3.6/site-packages/evidently/tabs/classification_performance_tab.py", line 8, in <module>
    from evidently.widgets.class_ref_quality_metrics_widget import ClassRefQualityMetricsWidget
  File "/usr/local/lib/python3.6/site-packages/evidently/widgets/class_ref_quality_metrics_widget.py", line 11, in <module>
    from sklearn import metrics
ModuleNotFoundError: No module named 'sklearn'
```

I've already solved this by installing `scikit-learn==0.23.2`.

So, I don't know if this is a real bug and scikit needs to be a dependency of evidently or not. It's up to you!

:smile:

**Comment by emeli-dral** [comment]
Hi @lfpelison , thank you for pointing this out! 
This is definitely a bug, will be fixed in the next package.

**Comment by emeli-dral** [comment]
fixed in ```evidently==0.1.15.dev0```
